src/sql/src/pure.rs

```diff
@@ -246,7 +246,8 @@ pub async fn purify_create_source(
     )
     .await?;
 
-    Ok(stmt)
+    let partial_stmt = crate::names::either_aug_raw(stmt);
+    Ok((vec![], partial_stmt))
```

Comment:
Am I missing something or does this always return an empty vector?

Reply:
This PR introduces the *ability* to return subsources but none of the sources return yet, so that's why it's empty